.github/workflows/bump-version-release.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
 
     steps:
       - name: Check out repository
-        uses: actions/checkout@v3
+        uses: bakdata/ci-templates/actions/checkout-repo@v1.26.0
```

Comment:
Did you test it? Your action is called `checkout` now instead of `checkout-repo`, right?

Reply:
its tested with `bakdata/ci-templates/actions/checkout@feat/checkout-repo`. so i believe it should work once the release is done after merge

Reply:
@MichaelKora You reverted this change again